src/whir/mod.rs

```diff
@@ -8,7 +8,7 @@ pub mod iopattern;
 pub mod parameters;
 pub mod prover;
 pub mod verifier;
-mod fs_utils;
+pub mod fs_utils;
```

Comment:
Why making this public? Are we using it in our own module?

Reply:
I copied it from the latest master branch, not sure why there is difference 😂

https://github.com/scroll-tech/whir/blob/main/src/whir/mod.rs#L11

Reply:
yeah, maybe...